nitro-protocol/gas-benchmarks/benchmark.test.ts

```diff
@@ -199,72 +205,100 @@ describe('Consumes the expected gas for sad-path exits', () => {
     ).toEqual(gasRequiredTo.ETHexitSadLedgerFunded.satp.total);
   });
 
-  // TODO unskip this test when the contracts are satp compatible
-  it.skip(`when exiting a virtual funded (with ETH) channel`, async () => {
+  it(`when exiting a virtual funded (with ETH) channel`, async () => {
     // begin setup
     await (
-      await nitroAdjudicator.deposit(MAGIC_ADDRESS_INDICATING_ETH, LforJ.channelId, 0, 10, {
-        value: 10,
-      })
+      await nitroAdjudicator.deposit(
+        // This deposit represents what in reality would likely be two deposits (one from Bob, one from Ingrid)
+        MAGIC_ADDRESS_INDICATING_ETH,
+        LforV.channelId,
+        0,
+        amountForAliceAndBob,
+        {
+          value: amountForAliceAndBob,
+        }
+      )
     ).wait();
     // end setup
-    // initially                   ⬛ ->  L  ->  J  ->  X  -> 👩
+    // initially                   ⬛ ->  L  ->  V  -> 👨
     // challenge L
     const {proof: ledgerProof, finalizesAt: ledgerFinalizesAt} = await challengeChannelAndExpectGas(
-      LforJ,
+      LforV,
       MAGIC_ADDRESS_INDICATING_ETH,
       gasRequiredTo.ETHexitSadVirtualFunded.satp.challengeL
     );
-    // challenge J
-    const {proof: jointProof, finalizesAt: jointChannelFinalizesAt} =
-      await challengeChannelAndExpectGas(
-        J,
-        MAGIC_ADDRESS_INDICATING_ETH,
-        gasRequiredTo.ETHexitSadVirtualFunded.satp.challengeJ
-      );
-    // challenge X
-    const {proof, finalizesAt} = await challengeChannelAndExpectGas(
-      X,
+
+    // challenge V with a voucher
+    const {
+      stateHash: vStateHash,
+      outcome: vOutcome,
+      gasUsed: vGasUsed,
+      finalizesAt: vFinalizesAt,
+    } = await challengeVirtualPaymentChannelWithVoucher(
+      V,
       MAGIC_ADDRESS_INDICATING_ETH,
-      gasRequiredTo.ETHexitSadVirtualFunded.satp.challengeX
+      BigNumber.from(paymentAmount).toNumber(),
+      Alice,
+      Bob
     );
+
+    expect(vGasUsed).toEqual(gasRequiredTo.ETHexitSadVirtualFunded.satp.challengeV);
+
     // begin wait
-    await waitForChallengesToTimeOut([ledgerFinalizesAt, jointChannelFinalizesAt, finalizesAt]);
+    await waitForChallengesToTimeOut([ledgerFinalizesAt, vFinalizesAt]);
     // end wait
-    // challenge L,J,X + timeout   ⬛ -> (L) -> (J) -> (X) -> 👩
+    // challenge L,V   + timeout   ⬛ -> (L) -> (V) -> 👨
```

Comment:
Is it explained anywhere what parentheses in (L) stand for?

Reply:
[c5ede49](https://github.com/statechannels/go-nitro/pull/831/commits/c5ede49843ed7e5c000c08b4f55c50897004bf90)